github-event/handler.go

```diff
@@ -168,7 +168,8 @@ func garbageCollect(garbageRequests []GarbageRequest) error {
 		if res.Body != nil {
 			defer res.Body.Close()
 		}
-		if res.StatusCode != http.StatusOK {
+		fmt.Printf("Status code for garbageCollect: %d\n", res.StatusCode)
```

Comment:
You can add 
`garbageRequest.Owner/Repo` so that the message can point to the GC-ed function